scripts/test/deploy.sh

```diff
@@ -25,8 +25,9 @@ INSTANCE_ID=$2
 VPC=$3
 SUBNET=$4
 DEPLOY_PAUSED=$5
+SPLIT_POINTS_FILE=$6
 
 SCRIPTS_DIR=$(cd "$(dirname "$0")" && cd .. && pwd)
 VERSION=$(cat "${SCRIPTS_DIR}/templates/version.txt")
 
-java -cp "${SCRIPTS_DIR}/jars/system-test-${VERSION}-utility.jar" sleeper.systemtest.cdk.DeployNewTestInstance "${SCRIPTS_DIR}" "${PROPERTIES_TEMPLATE}" "${INSTANCE_ID}" "${VPC}" "${SUBNET}" "${DEPLOY_PAUSED}"
+java -cp "${SCRIPTS_DIR}/jars/system-test-${VERSION}-utility.jar" sleeper.systemtest.cdk.DeployNewTestInstance "${SCRIPTS_DIR}" "${PROPERTIES_TEMPLATE}" "${INSTANCE_ID}" "${VPC}" "${SUBNET}" "${DEPLOY_PAUSED}" "${SPLIT_POINTS_FILE}"
```

Comment:
We're not handling the case where the split points file isn't specified. In that case `$6` will be unset, and it'll pass an empty string to Java here. The same thing will happen in the deployAll script.